### ISSUE TITLE
Default focus on login page 

### ISSUE THREAD

**Issue by Jessy-BAER** [BearStudio/start-ui-web#336]
I think this could be a good idea to add focus on field username by default ? 

![image](https://user-images.githubusercontent.com/7570753/219291847-c8e19d6c-c220-4238-a31f-2424874def89.png)

![image](https://user-images.githubusercontent.com/7570753/219292086-b61621b6-3bc1-48e0-83a6-d982087db817.png)

**Comment by yoannfleurydev** [comment]
I agree for the desktop part, for mobile, I'm not a huge fan of my keyboard popping without my consent.
Also, we have to make sure that it is OK for accessibility.

**Comment by Divyansh-0** [comment]
Can i take this up?

**Comment by yoannfleurydev** [comment]
> Can i take this up?

Hi sure, assigning you to this issue.

**Comment by Divyansh-0** [comment]
Hey i had a dobut Do you need yarn to install the dependencies because npm is giving me errors?

**Comment by yoannfleurydev** [comment]
> Hey i had a dobut Do you need yarn to install the dependencies because npm is giving me errors?

Yes, we are using `yarn` (v1) on this project, we do not use `npm`.

**Comment by Junaid300** [comment]
Is this issue fixed?  Can I work on this?

**Comment by yoannfleurydev** [comment]
> Is this issue fixed? Can I work on this?

Hey @Junaid300 , a PR is already opened for this issue https://github.com/BearStudio/start-ui-web/pull/406

**Comment by ivan-dalmet** [comment]
Old v2 issue. Reopen a new one if needed.
